Content/LuaScripts/ExampleCommonUIWidget.lua

```
@@ -0,0 +1,74 @@
+-- Example Common UI Widget Script
+-- This script demonstrates how to use Lua with Common UI widgets
+
+-- Called when the widget is constructed
+function OnWidgetConstructed(widget)
```

Luacheck (1.2.0)

[warning] setting non-standard global variable 'OnWidgetConstructed'

(W111)